src/main.ts

```diff
@@ -55,6 +87,10 @@ export default class Main {
     Main.application.on('window-all-closed', Main.onWindowAllClosed)
     Main.application.on('ready', Main.onReady)
 
+    app.on('session-created', (session) => {
```

Comment:
The `main` function stores 'app' to 'Main.application' so

```suggestion
    Main.application.on('session-created', (session) => {
```